src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/serviceprovider/performance/AuthUserJobParameter.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.reporting.serviceprovider.performance
+
+import org.springframework.batch.core.JobParameter
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUser
+
+data class AuthUserJobParameter(
+  val authUser: AuthUser
+) : JobParameter("") {
```

Comment:
Does this work?

The [documentation](https://www.docs4dev.com/javadoc/en/org/springframework/batch/spring-batch-core/4.2.4.RELEASE/org/springframework/batch/core/JobParameter.html) says

> Only the following types can be parameters: String, Long, Date, and Double

Reply:
Job arguments are likely serialised in https://hmpps-interventions-service-dev.apps.live-1.cloud-platform.service.justice.gov.uk/meta/schema/tables/batch_job_execution_params.html, which would make this difficult to work

Reply:
Passed in user parameters to construct authUSer instead